x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/edit_exception_flyout/index.tsx

```diff
@@ -367,6 +388,24 @@ const EditExceptionFlyoutComponent: React.FC<EditExceptionFlyoutProps> = ({
     prefix: 'exceptionFlyoutTitle',
   });
 
+  const confirmModal = useMemo(() => {
```

Comment:
Can this be moved out of this component and reused in add_exception_flyout.tsx ?